### PR TITLE
Hest test

### PR DESCRIPTION
**Pull request by sibo86**

### CODE REVIEW

File: collections/_web/v3/guides/simple_map.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ date: 2019-09-30
 
 ## Loading the Maps JavaScript API
 
-The Google Maps API are loaded using a script tag like so:
+The Google Maps API are loaded using a script tag like so: HEST
```
Comment (simonlaustsen) [inline collections/_web/v3/guides/simple_map.md:16]: Better like cows

Comment (sibo86) [reply collections/_web/v3/guides/simple_map.md]: fixed

**Review by simonlaustsen — APPROVED** [review]
👍

**Comment by sibo86** [comment]
testing

**Pull request closed without merging**